src/app/shared/components/sidemenu/sidemenu.component.spec.ts

```
@@ -228,13 +228,13 @@
     expect(teamOpt.length).toBe(2);
     expect(teamOpt).toEqual([
       {
-        title: 'Team expense reports',
+        title: 'Expense reports',
         isVisible:
           component.allowedActions.allowedReportsActions && component.allowedActions.allowedReportsActions.approve,
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)